src/TaglibSharp/Ogg/Bitstream.cs

```diff
@@ -119,14 +119,6 @@ public bool ReadPage (Page page)
 			ByteVector[] packets = page.Packets;
 
 			for (int i = 0; i < packets.Length; i++) {
-				if ((page.Header.Flags &
-					PageFlags.FirstPacketContinued) == 0 &&
-					previous_packet != null) {
-					if (ReadPacket (previous_packet))
-						return true;
-					previous_packet = null;
-				}
-
 
```

Comment:
This case can never happen any more - now we know whether a packet is complete we won't arrive here with a saved packet that turns out to be complete because `PageFlags.FirstPacketContinued` is not set